exporter/exporterhelper/internal/persistent_storage.go

```diff
@@ -173,7 +173,7 @@ func (pcs *persistentContiguousStorage) enqueueNotDispatchedReqs(reqs []Persiste
 	if len(reqs) > 0 {
 		errCount := 0
 		for _, req := range reqs {
-			if pcs.put(req) != nil {
+			if req == nil || pcs.put(req) != nil {
```

Comment:
When `req` is nil, there was some issue with unmarshaling the data. We should consider it an error. This change makes sure it is logged accordingly